DOCUMENTATION.md

```diff
@@ -749,13 +752,94 @@ is especially useful in rolling upgrade scenarios.
 
-The underlying format of the compact serialized objects is platform and language independent. All clients will have this feature
-in the future.
+### 4.1.1. Configuration
 
-### 4.1.1. Schema Evolution
+The only thing you need to configure is to register compact serializers. You can do that via
+`serialization.compact.serializers` configuration option.
+
+You have to supply a unique type name for the class in the serializer.
+
+Choosing a type name will associate that name with the schema and will make the polyglot use cases, where there are multiple
+clients from different languages, possible. Serializers in different languages can work on the same data, provided that their
+read and write methods are compatible, and they have the same type name.
+
+
+> **NOTES: If you evolve your class in the later versions of your application, by adding or removing fields, you should continue
+using the same type name for that class.**
+
+The following is an example of compact configuration:
+
+```js
+const client = await Client.newHazelcastClient({
+    serialization: {
+        compact: {
+            serializers: [new EmployeeSerializer()]
+        }
+    }
+});
+```
+
+### 4.1.2. Implementing CompactSerializer
+
+Compact serialization can be used by implementing a `CompactSerializer` for a class and registering it in the configuration.
+
+For example, assume that you have the following `Employee` class.
+
+```js
+class Employee {
+    constructor(age, id) {
+        this.age = age;
+        this.id = id;
+    }
+}
+```
+
+Then, a Compact serializer can be implemented as below.
+
+
+```js
+class EmployeeSerializer {
+    getTypeName() {
+        return 'Employee';
+    }
+
+    getClass() {
+        return Employee;
+    }
+
+    read(reader) {
+        const age = reader.readInt32('age');
+        const id = reader.readInt64('id');
+        return new Employee(age, id);
+    }
+
+    write(writer, value) {
+        writer.writeInt32('age', value.age);
+        writer.writeInt64('id', value.id);
+    }
+}
+```
+
+The last step is to register the serializer in the member or client configuration, as shown in the
+[Configuration](#411-configuration) section.
+
+Upon serialization, a schema will be created from the serializer, and a unique schema identifier will be assigned to it
+automatically.
+
+After the configuration registration, Hazelcast will serialize instances of the `Employee` class using the `EmployeeSerializer`.
+
+### 4.1.3. Supported Types
+
+Compact serialization supports the types in this list
+[in the reference manual](https://docs.hazelcast.com/hazelcast/latest/serialization/compact-serialization#supported-types)
+as first class types. Any other type can be implemented on top of these, by using these types as building blocks.
+
+**NOTE: Compact serialization supports circularly-dependent types, provided that the cycle ends at some point on runtime by some
```

Comment:
Please add '>' symbol.

`>**NOTE: Compact serialization supports circularly-dependent types, provided that the cycle ends at some point on runtime by some`

Reply:
https://github.com/hazelcast/hazelcast-nodejs-client/pull/1447/commits/d4ad872cecbdc3ef504f3ec4b654f61f54c74a5b